src/Common/Utility.hpp

```diff
@@ -294,6 +294,24 @@ namespace usbguard
     int _fd{-1};
   };
 
+
+  template<typename T>
+  std::string joinElements(T begin, T end, const std::string& separator = " ")
+  {
+    std::ostringstream ss;
+
+    if (begin != end) {
+      ss << *begin++;
```

Comment:
I think that *(begin++) is much  more readable.

Reply:
Okay

Reply:
I agree with @radosroka